_sass/blocks/_revenues_forms.scss

```diff
@@ -44,37 +44,13 @@
   }
 }
 
-.revenues-section-last {
-  padding-top: $base-padding-large;
-  padding-bottom: $base-padding-large;
+.revenues_page-forms_numbers {
```

Comment:
Selector `revenues_page-forms_numbers` should be written in lowercase with hyphens